package-lock.json

```diff
@@ -20512,4 +20512,4 @@
       }
     }
   }
-}
+}
```

Comment:
@gagan876574 this should not show as a diff. please add the newline. unless this file is removed from git diff i wont be able to merge.

Reply:
you can do
```
git checkout origin/master -- package-lock.json
```
and add the file and commit it.
this will revert the file to whatever is in master

Reply:
done that still showing diff

Reply:
@gagan876574 it now shows `+801 -1119` changes !! 
seems your master branch is not same as upstream master. so reset  hard your master branch to upstream/master and then if you do the above step, that should fix it.

Reply:
Done with the changes removed the packjson-lock.json diff